mono/metadata/appdomain.c

```diff
@@ -446,22 +446,26 @@ mono_context_init (MonoDomain *domain)
 void
 mono_context_init_checked (MonoDomain *domain, MonoError *error)
 {
+	HANDLE_FUNCTION_ENTER ();
```

Comment:
This could be moved down and still early return for no_exec but I went for consistency/idiom over micro optimization.